components/profile_popover/profile_popover.jsx

```diff
@@ -383,6 +403,64 @@ class ProfilePopover extends React.PureComponent {
             );
         }
 
+        if (this.props.isCustomStatusEnabled) {
+            let customStatusContent;
+            const customStatus = this.props.customStatus;
+            if (customStatus.text || customStatus.emoji) {
+                const customStatusEmoji = (
+                    <span className='d-flex'>
+                        <CustomStatusEmoji
+                            userID={this.props.user.id}
+                            showTooltip={false}
+                            emojiStyle={{
+                                marginRight: 4,
+                            }}
+                        />
+                    </span>
+                );
+
+                customStatusContent = (
+                    <div className='d-flex'>
+                        {customStatusEmoji}
+                        <div
+                            className='text-nowrap user-popover__email pb-1'
+                        >
+                            {customStatus.text}
+                        </div>
+                    </div>
+                );
+            } else if (this.props.user.id === this.props.currentUserId) {
+                customStatusContent = (
+                    <div>
+                        <a
+                            href='#'
+                            onClick={this.showCustomStatusModal}
+                        >
```

Comment:
1. Check if we should use something other than the `<a>` tag.
2. Add appropriate `aria` properties